WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteDialogFragment.kt

```diff
@@ -0,0 +1,101 @@
+package org.wordpress.android.ui.people
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.fragment.app.DialogFragment
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.lifecycle.ViewModelStoreOwner
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.wordpress.android.R
+import org.wordpress.android.WordPress
+import org.wordpress.android.ui.people.PeopleInviteDialogFragment.DialogMode.DISABLE_INVITE_LINKS_CONFIRMATION
+import org.wordpress.android.ui.people.PeopleInviteDialogFragment.DialogMode.INVITE_LINKS_ROLE_SELECTION
+import org.wordpress.android.viewmodel.ContextProvider
+import javax.inject.Inject
+
+class PeopleInviteDialogFragment : DialogFragment() {
```

Comment:
Since this is a multipurpose dialog, maybe we should add a comment explaining what it does.